airbyte-integrations/connectors/source-kafka/src/main/java/io/airbyte/integrations/source/kafka/converter/Converter.java

```diff
@@ -0,0 +1,8 @@
+package io.airbyte.integrations.source.kafka.converter;
+
+import io.airbyte.protocol.models.v0.AirbyteMessage;
+
+public interface Converter<V> {
+
+  AirbyteMessage convertToAirbyteRecord(String topic, V value);

```

Comment:
Make this return the most specific type, AirbyteRecordMessage and create the AirbyteMessage in the generator.